src/manage.c

```diff
@@ -4031,7 +4039,17 @@ get_osp_performance_string (scanner_t scanner, int start, int end,
   opts.titles = g_strdup (titles);
   error = NULL;
 
-  if (osp_get_performance_ext (connection, opts, performance_str, &error))
+  connection_retry = SCANNER_CONNECTION_RETRY_DEFAULT;
+  int return_value = 1;
```

Comment:
Could you move the declaration to the top of the function and only do the initialization here?